src/templates/en/2019/base_chapter.html

```diff
@@ -32,6 +32,14 @@
   var el = document.getElementById('num_comments');
   el.innerText = comments + ' ' + (comments == 1 ? 'comment' : 'comments');
 });
+document.addEventListener("keypress", function onPress(event) {
+ 	if (event.key === 'p' || event.key === ',') {
+ 		document.getElementById('previous-chapter').click();
```

Comment:
Need to handle the edge cases for the first and last chapters, for example:

```suggestion
 		var previous = document.getElementById('previous-chapter').click();
 		if (previous) {
 		  previous.click();
 		}
```